docs/index.rst

```diff
@@ -15,6 +15,9 @@ standalone or Kubernetes-based application gateway.
    This project is under active development. Many, many features are not
    complete. We would love for you to :doc:`get involved<get_involved>`.
 
+   This project is now listed in official 
```

Comment:
s/in official/as an official/

IMO this information is better conveyed on the compatibility page. Thoughts @arkodg @LukeShu @skriss @AliceProxy?